packages/eslint-plugin-next/lib/index.js

```diff
@@ -1,60 +1,60 @@
 module.exports = {
   rules: {
-    'no-css-tags': require('./rules/no-css-tags'),
-    'no-sync-scripts': require('./rules/no-sync-scripts'),
-    'no-html-link-for-pages': require('./rules/no-html-link-for-pages'),
-    'no-img-element': require('./rules/no-img-element'),
-    'no-head-element': require('./rules/no-head-element'),
-    'no-unwanted-polyfillio': require('./rules/no-unwanted-polyfillio'),
-    'no-page-custom-font': require('./rules/no-page-custom-font'),
-    'no-title-in-document-head': require('./rules/no-title-in-document-head'),
     'google-font-display': require('./rules/google-font-display'),
     'google-font-preconnect': require('./rules/google-font-preconnect'),
+    'inline-script-id': require('./rules/inline-script-id'),
+    'next-script-for-ga': require('./rules/next-script-for-ga'),
+    'no-before-interactive-script-outside-document': require('./rules/no-before-interactive-script-outside-document'),
+    'no-css-tags': require('./rules/no-css-tags'),
     'no-document-import-in-page': require('./rules/no-document-import-in-page'),
+    'no-duplicate-head': require('./rules/no-duplicate-head'),
+    'no-head-element': require('./rules/no-head-element'),
     'no-head-import-in-document': require('./rules/no-head-import-in-document'),
+    'no-html-link-for-pages': require('./rules/no-html-link-for-pages'),
+    'no-img-element': require('./rules/no-img-element'),
+    'no-page-custom-font': require('./rules/no-page-custom-font'),
     'no-script-component-in-head': require('./rules/no-script-component-in-head'),
     'no-server-import-in-page': require('./rules/no-server-import-in-page'),
     'no-styled-jsx-in-document': require('./rules/no-styled-jsx-in-document'),
+    'no-sync-scripts': require('./rules/no-sync-scripts'),
+    'no-title-in-document-head': require('./rules/no-title-in-document-head'),
     'no-typos': require('./rules/no-typos'),
-    'no-duplicate-head': require('./rules/no-duplicate-head'),
-    'inline-script-id': require('./rules/inline-script-id'),
-    'next-script-for-ga': require('./rules/next-script-for-ga'),
-    'no-before-interactive-script-outside-document': require('./rules/no-before-interactive-script-outside-document'),
-    'no-assign-module-variable': require('./rules/no-assign-module-variable'),
```

Comment:
Was this rule removed on purpose? The file still exists. 🤔

Reply:
@mrtnzlml no, it was not removed on purpose. It was an oversight on my part after resolving many merge conflicts. I appreciate you calling it out. I'll look over the rest to ensure no others were missed and whip up another PR to fix this. Thank you!

Reply:
https://github.com/vercel/next.js/pull/38134

Reply:
Cool! 👍